src/backend/adapters/WishListAdapter.ts

```diff
@@ -0,0 +1,80 @@
+import { DbWishItem, DbWishList } from "../domain/models/WishList"
```

Comment:
this does the storage & retrieval of wish lists from db.  It will not contain business logic, just methods for write/reading the basic entities.

I'll be putting business logic in the WishListService later;  it'll probably have rules on "who can edit what" - because that's business rules - (you can't edit my wish list).  Or it'll have rules about reserving a wish-list item (you must be signed in).  Things like that.